Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm

```diff
@@ -816,6 +835,19 @@ - (void)_showPhotoPickerWithSourceType:(UIImagePickerControllerSourceType)source
     [self _presentFullscreenViewController:_imagePicker.get() animated:YES];
 }
 
+- (void)_showPhotoPicker
+{
+    auto config = adoptNS([allocPHPickerConfigurationInstance() init]);
```

Comment:
Nit: `configuration`.

We generally prefer to avoid abbreviations / short forms in the code.